src/nvim/types.h

```diff
@@ -13,4 +13,5 @@ typedef unsigned char char_u;
 // Can hold one decoded UTF-8 character.
 typedef uint32_t u8char_T;
 
+typedef struct expand expand_T;
```

Comment:
Is this needed for the test? can you `helpers.cimport("./src/nvim/ex_cmds_defs.h")`  instead?